compass/landice/tests/ismip6_run/ismip6_ais_proj2300/set_up_experiment.py

```diff
@@ -0,0 +1,254 @@
+import glob
+import os
+import shutil
+import sys
+
+from compass.io import symlink
+from compass.job import write_job_script
+from compass.model import make_graph_file, run_model
+from compass.step import Step
+
+
+class SetUpExperiment(Step):
+    """
+    A step for setting up an ISMIP6 experiment
+
+    Attributes
+    ----------
+    """
+
+    def __init__(self, test_case, name, subdir, exp):
+        """
+        Set up a new experiment
+
+        Parameters
+        ----------
+
+        exp : experiment
+        """
+
+        self.exp = exp
+
+        super().__init__(test_case=test_case, name=name, subdir=subdir)
+
+    def setup(self):  # noqa:C901
+
+        print(f"    Setting up experiment {self.exp}")
+
+        config = self.config
+        section = config['ismip6_run_ais_2300']
+        mesh_res = section.getint('mesh_res')
+        self.ntasks = section.getint('ntasks')
+        self.min_tasks = self.ntasks
+        forcing_basepath = section.get('forcing_basepath')
+        init_cond_path = section.get('init_cond_path')
+        init_cond_fname = os.path.split(init_cond_path)[-1]
+        melt_params_path = section.get('melt_params_path')
+        melt_params_fname = os.path.split(melt_params_path)[-1]
+        region_mask_path = section.get('region_mask_path')
+        region_mask_fname = os.path.split(region_mask_path)[-1]
+        calving_method = section.get('calving_method')
+
+        if self.exp == 'hist':
+            exp_fcg = 'ctrlAE'
+        else:
+            exp_fcg = self.exp
+
+        # Define where to get namelist, streams, yaml, etc. templates
+        # (in current package)
+        resource_location = \
+            'compass.landice.tests.ismip6_run.ismip6_ais_proj2300'
+
+        # Define calving method
+        if calving_method == 'von_mises':
+            use_vM_calving = True
+        elif calving_method == 'restore':
+            use_vM_calving = False
+
+        # Figure out if the forcing is in tier1 or tier2 subdir
+        if 'exp' in self.exp:
+            if int(self.exp[-2:]) >= 7:
+                forcing_basepath = os.path.join(forcing_basepath,
+                                                'tier2_experiments')
+            else:
+                forcing_basepath = os.path.join(forcing_basepath,
+                                                'tier1_experiments')
+        else:
+            forcing_basepath = os.path.join(forcing_basepath,
+                                            'tier1_experiments')
+
+        # Copy files we'll need from local paths specified in cfg file
+        if self.exp == 'hist':
+            shutil.copy(init_cond_path, self.work_dir)
+        shutil.copy(melt_params_path, self.work_dir)
+        shutil.copy(region_mask_path, self.work_dir)
+
+        # Find and copy correct forcing files
+        smb_search_path = os.path.join(
+            forcing_basepath, exp_fcg,
+            'processed_SMB_*_smbNeg_over_bareland.nc')
```

Comment:
Thanks for taking over the PR and making it look/work fantastic, @matthewhoffman!  During the final review step of the `ismip6_forcing` test group, a change has been made that this forcing file name is not relevant anymore. That is, the `processed_` part was replaced with `mali mesh name`, e.g. AIS_4to20km_r01. The (https://github.com/hollyhan/compass/blob/867bab78c3b8af109069c571acc9c87a02885fe3/compass/landice/tests/ismip6_forcing/atmosphere/process_smb_racmo.py#L53-L54).

Reply:
I created an issue here: https://github.com/MPAS-Dev/compass/issues/589